0000-template.md

```diff
@@ -14,26 +14,23 @@ One paragraph explanation of the proposal.
 
 ## Motivation
 
-Why are we doing this? What use cases does it support? What problems does it solve? What is the
-expected outcome?
+- Why are we doing this?
+- What use cases does it support?
+- What problems does it solve?
+- What is the expected outcome?
 
 ## Stakeholders
 
-- Who is affected by this HIP? A stakeholder is any individual, group, or party that has an interest
-  in an organization and the outcomes of its actions.
-
+- Who is affected by this HIP? A stakeholder is any individual, group, or party such as network
+  users, Hotspot hosts, or token holders.
```

Comment:
"party that has an interest in an organization and the outcomes of its actions" feels unclear to me. 
This sentence should clarify "Who is affected by the HIP", hence a couple of examples proposed.